demo/src/diagram-viewers/add-diagrams.ts

```diff
@@ -399,6 +399,13 @@ const handleNextVL: OnNextVoltageCallbackType = (id: string) => {
     console.log(msg);
 };
 
+const handleNextVLWithEvent: OnNextVoltageCallbackType = (id: string, event: MouseEvent) => {
+    const msg = event.ctrlKey
+        ? 'Clicked on CTRL and navigation arrow, dest VL is ' + id
+        : 'Clicked on navigation arrow, dest VL is ' + id;
+    console.log(msg);
+};
+
```

Comment:
Maybe you could just edit `handleNextVL` to implement `OnNextVoltageCallbackType` with the event parameter instead of duplicate definition ?
something strange in the text : "Clicked on CTRL and navigation arrox..."
Maybe replace with something like "CTRL + Click on navigation arrow, ..."